src/components/ModifiersGroup.vue

```diff
@@ -41,12 +46,44 @@ export default {
       return formatedPrice
     },
   },
+  methods: {
+    ...mapMutations('modifier', ['setCurrentMenuModifier']),
+    upDateModifiers(newModifiers, oldModifiers) {
+      let currentModifiers = this.getCurrentMenuModifier
+      if (oldModifiers) {
+        let modifierIds = []
+        if (_d.isArray(oldModifiers)) modifierIds = _d.map(oldModifiers, 'id')
+        else modifierIds = [oldModifiers.id]
+        _d.remove(currentModifiers, ({id}) => {
+          return modifierIds.includes(id)
+        })
+      }
+
+      let newFormatedModifier = []
+      if (_d.isArray(newModifiers)) {
+        newFormatedModifier = _d.map(this.defaultModifiers, ({id, price}) => ({
+          id,
+          price,
+        }))
+      } else newFormatedModifier = [_d.pick(this.defaultModifiers, ['id', 'price'])]
+
+      this.setCurrentMenuModifier(_d.concat(currentModifiers, newFormatedModifier))
+    },
+  },
   mounted() {
     let modifierDedault = _d.find(this.data, modifier => {
       return modifier.isDefault
     })
     if (modifierDedault) this.defaultModifiers = modifierDedault
```

Comment:
this.defaultModifiers =_d.find(this.data, modifier => {
             return modifier.isDefault
  })